### PR TITLE
feat: create a new community

### LINKED ISSUE #10: U06: As a registered user, I want to create a new (non pre-existing) course community so that I and other students can rate this course and upload post to the course page.
Requirement: The social network should allow a user to create new (non-pre-existing) course communities.

# Development Tasks

- [ ] The-Agile-Avengers/icu-connect-frontend#32
- [ ] The-Agile-Avengers/icu-connect-frontend#33
- [ ] The-Agile-Avengers/icu-connect-backend#6
- [ ] The-Agile-Avengers/icu-connect-backend#43

### PR DESCRIPTION
**Pull request by Mariinja**
closes #10

### CODE REVIEW

File: src/components/course/CommunityCreate.tsx [hunk 1]
```diff
@@ -80,42 +72,71 @@ export default function CommunityCreate() {
       direction="column"
       justifyContent="space-evenly"
       alignItems="center"
-      spacing={2}
+      spacing={4}
+      sx={{
+        width: "100%",}}
```
Comment (Taremeh) [inline src/components/course/CommunityCreate.tsx:77]:
```suggestion
      spacing={4}
      sx={{width: "100%"}}
```

**Review by Taremeh — CHANGES_REQUESTED** [review]
The component is wrapped by a `Layout` Component (line 70) in the `CommunityCreate.tsx` file and wrapped again in a `Layout` Component in the `Community.tsx` file (line 277). Considering, that for we've wrapped the components in `Community.tsx` for every other if/else case, it would make sense to remove the Layout wrapping here. If React complains due to the single child element constraint, you can instead wrap it using "<> .... </>" which is common practice in React.

File: src/components/course/CreateCommunityForm.tsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+export default function CommunityCreate() {
+  const navigate = useNavigate();
+
+
+  const { register, handleSubmit , formState} = useForm<formValues>({
+    mode: "onChange"
+  });
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const onSubmit = (formData: any) => {
+    console.log(formData)
```
Comment (StefanieKrohmann) [inline src/components/course/CreateCommunityForm.tsx:49]: duplicate console.log()

File: src/components/course/CreateCommunityForm.tsx [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  console.log(formData)
```
Comment (StefanieKrohmann) [inline src/components/course/CreateCommunityForm.tsx:50]: duplicate console.log()

**Review by StefanieKrohmann — DISMISSED** [review]
👍

**Review by StefanieKrohmann — APPROVED** [review]

**Review by Taremeh — APPROVED** [review]
👍👍👍

**Pull request merged**